docs/server/node/upgrade_guide/v5ToV6UpgradeGuide.mdx

```diff
@@ -0,0 +1,108 @@
+---
+slug: /server/nodejsServerSDK/upgrade_guide/v5ToV6
+title: Node SDK v5.x.x to v6.0.0 Upgrade Guide
+---
+### Summary 
+
+Node.js SDK V6.0.0 introduced breaking changes on 
+- Removed async version of core APIs (checkGate, getConfig..)
+- Changed DataAdapter cache keys SDK used to read and write from
+- Changed default hashing algorithm used for getClientInitializeResponse() which is being used for bootstrapping client sdk.
+- If no environment tier is being set for SDK, we will default environment to be `production` when evaluating. 
+- Changed how you should disable exposure logging
```

Comment:
each of these should link to a section below that explains it in more detail.  So 

`#### Synchronous Core APIs` is a good start, but you need one for each line